### PR TITLE
Extracted search and web-worker utilities into separate js-worker-search library

### PR DESCRIPTION
**Pull request by bvaughn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -73,12 +75,10 @@
...
     "watch": "^0.16.0",
     "webpack": "^1.12.9",
-    "webpack-dev-server": "^1.14.0",
-    "worker-loader": "^0.7.0"
+    "webpack-dev-server": "^1.14.0"
   },
   "dependencies": {
-    "node-uuid": "^1.4.7",
-    "react-highlight-words": "^0.1.1",
+    "js-worker-search": "file:///Users/brianvaughn/Documents/git/js-worker-search/js-worker-search-1.0.0.tgz",
```
Comment (cesarandreu) [inline package.json]: ???

File: package.json [hunk 2]
```diff
@@ -59,9 +59,11 @@
     "gh-pages": "^0.8.0",
     "html-webpack-plugin": "^1.7.0",
     "immutable": "^3.7.5",
+    "js-worker-search": "file:///Users/brianvaughn/Documents/git/js-worker-search/js-worker-search-1.0.0.tgz",
```
Comment (cesarandreu) [inline package.json]: ???

File: package.json (hunk 1 above)
Comment (bvaughn) [inline package.json]: Hurr durr. Sorry. I did not mean to commit this version.

**Comment by bvaughn** [comment]
Fixed the accidental package flub. My bad.

**Comment by bvaughn** [comment]
Bump @cesarandreu :)

**Comment by bvaughn** [comment]
Bumping @cesarandreu

**Pull request merged**